website/client/main.js

```diff
@@ -9,6 +9,7 @@ import getStore from './store';
 import StoreModule from './libs/store';
 import './filters/registerGlobals';
 import i18n from './libs/i18n';
+import 'smartbanner.js/dist/smartbanner';
```

Comment:
instead of loading the version in /dist which is already compiled using browserify we should load the original version that will then be compiled directly in Habitica, so the file to import should be `smartbanner.js/src/index.js`

Reply:
Hmm, the /src/ directory isn't included in the node module only /dist/ which containts smartbanner.js and smartbanner.min.js

Reply:
I see... In that case the non .min version will work

Reply:
Shouldn't we be able to just import smartbanner.js? It is a module

Reply:
Looks like it's mostly published to npm but doesn't follow many conventions and if I remember correctly it doesn't have a main entry in package.json so i don't think it'll work, but we can try

Reply:
Updated - just committed the change